tests/sensors/test_hdfs_sensor.py

```diff
@@ -22,7 +22,7 @@
 from airflow.exceptions import AirflowSensorTimeout
 from airflow.sensors.hdfs_sensor import HdfsSensor
 from airflow.utils.timezone import datetime
-from tests.core import FakeHDFSHook
+from tests.test_utils.hdfs_utils import FakeHDFSHook
```

Comment:
Maybe it's worth to consider to make `tests.test_utils.mocks` to keep there all fake / mock stuff?

Reply:
If there are a large number of them, we can create such a structure. Now it would only complicate the situation/import.

Reply:
If someone wants to add more things to HDFS then they will have ready place.